test/python/test_onnxruntime_genai_e2e.py

```diff
@@ -53,5 +53,7 @@ def run_model(model_path: str | bytes | os.PathLike):
     for model_name in ["microsoft/phi-2"]:
         with tempfile.TemporaryDirectory() as temp_dir:
             device = "cuda" if og.is_cuda_available() else "cpu"
-            download_model(temp_dir, device, model_name)
+            download_model(temp_dir, device, model_name, int4)
```

Comment:
I am a little confused. How does python know that `int4` is a `str`? Should it not be `"int4"`?

Reply:
Fixed